website/docs/Installation.md

```diff
@@ -31,6 +31,27 @@ The following command will deactivate the current `conda` environment:
 conda deactivate
 ```
 
+### Option 3: poetry
+
+Another option is with `poetry`, which is a dependency manager for Python. 
+
+[Poetry is a tool for dependency management and packaging in Python. It allows you to declare the libraries your project depends on and it will manage (install/update) them for you. Poetry offers a lockfile to ensure repeatable installs, and can build your project for distribution.
+](https://python-poetry.org/docs/)
+
+You can install it by following [this doc](https://python-poetry.org/docs/#installation),
+and then create a virtual environment as below:
+```bash
+poetry init
+poetry shell
+
+poetry add pyautogen
+```
```

Comment:
Add a few words here, maybe:
```
It will generate a file named `pyproject.toml` under your working directory and create a virtual environment under
`<YOUR-HOME-DIRECTORY>/.cache/pypoetry/virtualenvs/`. You'll have a separate virtual environment for each of
your projects. This is different from conda, which allows you to reuse a virtual environment for different projects.
```

Please correct me if my understanding is wrong.

Reply:
The words may or may not be needed. Keeping autogen's doc isolated about what it is about seems like a better idea for me. If anybody wants to learn about poetry, they have a good reference to the poetry docs already. It may seem like too much information from my perspective as a developer. 

I'm also assuming those who are used to using poetry will understand how poetry works and choose this path.